geode-core/src/integrationTest/java/org/apache/geode/cache/query/internal/index/IndexMaintenanceJUnitTest.java

```diff
@@ -1057,6 +1071,394 @@ public void testBug43597() throws Exception {
     region.put("1", pf1);
   }
 
+  /**
+   * Starting of section - moved from wrongly spelled old class "IndexMaintainceJUnitTest".
+   * GEODE-1505
+   */
+
+  @Test
+  public void test000BUG32452()
```

Comment:
This test might be better named "fromClausesAndIndexedExpressionsMatchExpectedValues"